Java/tests/src/test/java/com/microsoft/recognizers/text/tests/datetime/DateTimeExtractorTest.java

```diff
@@ -124,71 +141,102 @@ private static IDateTimeExtractor getEnglishExtractor(String name) {
-                return new BaseDatePeriodExtractor(new EnglishDatePeriodExtractorConfiguration(config));
-            //case "DateTimeAltExtractor":
-            //    return new BaseDateTimeAltExtractor(new EnglishDateTimeAltExtractorConfiguration());
-            case "DateTimeExtractor":
-                return new BaseDateTimeExtractor(new EnglishDateTimeExtractorConfiguration());
-            case "DateTimePeriodExtractor":
-                return new BaseDateTimePeriodExtractor(new EnglishDateTimePeriodExtractorConfiguration());
-            case "DurationExtractor":
-                return new BaseDurationExtractor(new EnglishDurationExtractorConfiguration());
-            case "HolidayExtractor":
-                return new BaseHolidayExtractor(new EnglishHolidayExtractorConfiguration());
-            case "MergedExtractor":
-                return new BaseMergedDateTimeExtractor(new EnglishMergedExtractorConfiguration(DateTimeOptions.None));
-            case "MergedExtractorSkipFromTo":
-                return new BaseMergedDateTimeExtractor(new EnglishMergedExtractorConfiguration(DateTimeOptions.SkipFromToMerge));
-            case "SetExtractor":
-                return new BaseSetExtractor(new EnglishSetExtractorConfiguration());
-            case "TimeExtractor":
-                return new BaseTimeExtractor(new EnglishTimeExtractorConfiguration());
-            case "TimePeriodExtractor":
-                return new BaseTimePeriodExtractor(new EnglishTimePeriodExtractorConfiguration());
-            case "TimeZoneExtractor":
-                return new BaseTimeZoneExtractor(new EnglishTimeZoneExtractorConfiguration(DateTimeOptions.EnablePreview));
-
-            default:
-                throw new AssumptionViolatedException("Extractor Type/Name not supported.");
+        case "DateExtractor":
+            return new BaseDateExtractor(new EnglishDateExtractorConfiguration(config));
+        case "DatePeriodExtractor":
+            return new BaseDatePeriodExtractor(new EnglishDatePeriodExtractorConfiguration(config));
+        //case "DateTimeAltExtractor":
+        //    return new BaseDateTimeAltExtractor(new EnglishDateTimeAltExtractorConfiguration());
+        case "DateTimeExtractor":
+            return new BaseDateTimeExtractor(new EnglishDateTimeExtractorConfiguration());
+        case "DateTimePeriodExtractor":
+            return new BaseDateTimePeriodExtractor(new EnglishDateTimePeriodExtractorConfiguration());
+        case "DurationExtractor":
+            return new BaseDurationExtractor(new EnglishDurationExtractorConfiguration());
+        case "HolidayExtractor":
+            return new BaseHolidayExtractor(new EnglishHolidayExtractorConfiguration());
+        case "MergedExtractor":
+            return new BaseMergedDateTimeExtractor(new EnglishMergedExtractorConfiguration(DateTimeOptions.None));
+        case "MergedExtractorSkipFromTo":
+            return new BaseMergedDateTimeExtractor(
```

Comment:
Nitpick: Please don't change previous formatting unless an error was being raised. Can you revert?